yajsapi/rest/payment.ts

```diff
@@ -382,9 +382,9 @@ export class Payment {
           "getDebitNoteEvents"
         );
         for (const ev of events) {
-          self.logger?.debug(
-            `Received debit note event: ${JSON.stringify(ev)}, ` + `type: ${JSON.stringify(Object.getPrototypeOf(ev))}`
-          );
+          // self.logger?.debug(
```

Comment:
Is it consciously commented out?

Reply:
Yes, temporary I disabled this because it caused bug described in #381